src/Commands/Backup/ListCommand.php

```diff
@@ -22,6 +22,7 @@ class ListCommand extends BackupCommand
      *     file: Filename
      *     size: Size
      *     date: Date
+     *     expiry: Expiry
```

Comment:
While I personally like this, I believe US english prefers 'expiration'. But I'm  1/10 on this because expiry is shorter and way classier because it is true uncorrupted queens english :)